src/engraving/layout/layoutmeasure.cpp

```diff
@@ -880,7 +880,6 @@ void LayoutMeasure::computePreSpacingItems(Measure* m)
             isFirstChordInMeasure = false;
 
             chord->layoutArticulations();
-            chord->layoutArticulations2();
```

Comment:
Ideally this should be refactored so you do not need to call two different methods to lay out the articulations.  It seems odd to need to lay them out twice.